README.md

```diff
@@ -1 +1,64 @@
-# Hello-world
\ No newline at end of file
+
+# 📖 Hello world project
+
+> This is a hello world microverse project
+
+## 🛠 Built With:
+- Major languages
+ -Html
+ -css
+
+### Tech Stack 
+- Technologies used
+ 1 .Git, Github and linters
+
+## 💻 Getting Started 
+
+> To get started, you need to visit the gihub repository at https://github.com/ismayito/Hello-world
+
+To get a local copy up and running, follow these steps.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+That is: 
+-open your command line 
+-navigate to your desired folder by typing cd my-folder on the terminal
+-then type  git clone https://github.com/ismayito/Hello-world.git.
+
+### Install
+-After downloading you can extract the zipped files to your folder.
+-open the index.html file and run it in the browser.
+-You will have your project up and running;
+
+
+### Usage
+
+To run the project:
+Open the project in your favourite browser
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author: Mayito Ismail**
+
+- GitHub: [@githubhandle](https://github.com/ismayito)
+- Twitter: [@twitterhandle](https://twitter.com/@IsmailMayito)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mayito-ismail-2b0067178/)
```

Comment:
- [OPTIONAL] Good job putting the links to your social accounta, I think instead of `@githubhandle` and others you could put your true handles to signify to the user that your links are really present :+1: